index.js

```diff
@@ -122,21 +138,21 @@ module.exports = {
     strict: 0,
     'object-curly-spacing': 0,
     'operator-linebreak': [1, 'after'],
-    quotes: [0, 'single', 'avoid-escape'],
+    quotes: [1, 'single', 'avoid-escape'],
```

Comment:
oh interesting - seems fine to me as long as people know.

i remember us deciding at some point to not quote object keys, but then running into kebab-case things. as long as it's made explicit that we are doing this then 👍